containers/tools/deploy.yml

```diff
@@ -12,21 +12,12 @@
         name: epel-release
         state: present
 
-    - name: 'Install pip'
-      package:
-        name: python-pip
-        state: present
-
     - name: 'Update setuptools'
       command: 'pip install setuptools --upgrade'
 
-    - name: 'Install python openshift'
-      command: pip install git+https://github.com/ehelms/openshift-restclient-python.git@fix-destination-ca
-
-    - name: 'Install kubernetes modules'
-      command: 'ansible-galaxy install -r requirements.yml'
-      args:
-        chdir: "{{ working_dir }}"
+    - name: 'Install openshift'
+      pip:
+        name: openshift
```

Comment:
The ansible module [docs](https://docs.ansible.com/ansible/latest/modules/openshift_raw_module.html#openshift-raw-module) list a rather strict version requirement for this package (`== 0.4.3`). Can we (and do we want to) enforce that here?

Reply:
I noticed that too, and I've had no issues with newer versions. Further, newer versions handle some more advanced options better (such as the certificate_ca_file) so I am inclined to ride latest.